applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.cpp

```diff
@@ -1353,6 +1363,9 @@ void SmallStrainUPwDiffOrderElement::CalculateMaterialStiffnessMatrix(MatrixType
         Variables.F                  = deformation_gradients[GPoint];
         Variables.StrainVector       = strain_vectors[GPoint];
         Variables.ConstitutiveMatrix = constitutive_matrices[GPoint];
+        ConstitutiveLawUtilities::SetConstitutiveParameters(
```

Comment:
This function call is no longer necessary, since the constitutive matrices are already calculated (they are only retrieved here from the vector).

Reply:
removed